app/templates/partials/index/navigation.html

```diff
@@ -30,6 +30,15 @@
         <li class="nav-item">
           <a class="nav-link" href="/search">Search</a>
         </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/celebrity">Celebrities Born Today</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="{{ url_for('salary_home') }}">Salary</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="{{ url_for('meds') }}">Medication</a>
```

Comment:
I think it is more suited for the feature panal by @aviadamar what do you think?

Reply:
I reckon all of the links in this section should eventually transfer to the feature panel, but since it is currently only visible on one page, I put the link here with the rest of the features.